lib/matplotlib/transforms.py

```diff
@@ -1418,24 +1418,28 @@ def contains_branch(self, other):
                 return True
         return False
 
-    def contains_branch_seperately(self, other_transform):
+    def contains_branch_separately(self, other_transform):
         """
         Return whether the given branch is a sub-tree of this transform on
         each separate dimension.
 
         A common use for this method is to identify if a transform is a blended
         transform containing an Axes' data transform. e.g.::
 
-            x_isdata, y_isdata = trans.contains_branch_seperately(ax.transData)
+            x_isdata, y_isdata = trans.contains_branch_separately(ax.transData)
 
         """
         if self.output_dims != 2:
-            raise ValueError('contains_branch_seperately only supports '
+            raise ValueError('contains_branch_separately only supports '
                              'transforms with 2 output dimensions')
         # for a non-blended transform each separate dimension is the same, so
         # just return the appropriate shape.
         return (self.contains_branch(other_transform), ) * 2
 
+    def contains_branch_seperately(self, other_transform):
+        """Permanent alias for backwards compatibility (historical typo)."""
```

Comment:
If you set this private, then it shouldn't be documented (as I believe was decided in #30474)
```suggestion
    # Permanent alias for backwards compatibility (historical typo).
    def contains_branch_seperately(self, other_transform):
        """:meta private:"""
```

Reply:
Great point! Done.